src/components/AccountLoginForm.jsx

```diff
@@ -5,7 +5,7 @@ import classNames from 'classnames'
 import statefulForm from '../lib/statefulForm'
 import Field, { PasswordField, DropdownField, CheckboxField } from './Field'
 
-const AccountLoginForm = ({ t, konnector, customView, fields, error, dirty, submitting, values, submit, onCancel }) => {
+const AccountLoginForm = ({ t, konnector, customView, fields, error, dirty, submitting, deleting, values, submit, onDelete, onCancel }) => {
   const isUpdate = !!values.login || !!values.access_token
```

Comment:
You're right, that was its precedent name. My point was that as we are in a form we just need to know if it is a creation or an update, not if the account is connected. For example, if the credentials are wrong, you could update the values without having an actually "connected" account.